CSharp/DevVmPowershell/Helpers/IApplicationInstallHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Helpers
+{
+	public interface IApplicationInstallHelper
+	{
+		bool InstallApplicationFromRapFile(int workspaceId, string filePath);
+		bool InstallApplicationFromApplicationLibrary(int workspaceId, string applicationGuid);
```

Comment:
Can we rename it to ApplicationFromLibrary to keep it simple. Two Application words seems weird.